projects/plugins/jetpack/modules/custom-css/custom-css.php

```diff
@@ -371,14 +376,16 @@ public static function customize_controls_enqueue_scripts() {
 				/** This filter is documented in modules/custom-css/custom-css.php */
 				'useRichEditor'        => ! jetpack_is_mobile() && apply_filters( 'safecss_use_ace', true ),
 				'areThereCssRevisions' => self::are_there_css_revisions(),
+				'startFresh'           => isset( $setting ) && $setting->value(),
 				'revisionsUrl'         => self::get_revisions_url(),
 				'cssHelpUrl'           => '//en.support.wordpress.com/custom-design/editing-css/',
 				'l10n'                 => array(
-					'mode'           => __( 'Start Fresh', 'jetpack' ),
-					'mobile'         => __( 'On Mobile', 'jetpack' ),
-					'contentWidth'   => $content_help,
-					'revisions'      => _x( 'See full history', 'Toolbar button to see full CSS revision history', 'jetpack' ),
-					'css_help_title' => _x( 'Help', 'Toolbar button to get help with custom CSS', 'jetpack' ),
+					'mode'                        => __( 'Start Fresh', 'jetpack' ) . ' (' . __( 'deprecated', 'jetpack' ) . ')',
+					'mobile'                      => __( 'On Mobile', 'jetpack' ),
+					'contentWidth'                => $content_help,
+					'revisions'                   => _x( 'See full history', 'Toolbar button to see full CSS revision history', 'jetpack' ),
+					'css_help_title'              => _x( 'Help', 'Toolbar button to get help with custom CSS', 'jetpack' ),
+					'startFreshCustomizerWarning' => __( "The Start Fresh option in the Additional CSS panel is enabled, which means the theme's original CSS is not applied. This option will no longer be supported.", 'jetpack' ),
```

Comment:
With this, along with `StartFreshDeprecationWarning`, I wonder if we may need design/marketing input on the wording itself here? The part I am stumbling on in particular is 'This option will no longer be supported.' - I wonder if we need to allude to a sense of urgency even if not giving a date.

Reply:
Yes, I came here to suggest the same thing. I think we need a date for it to be really useful. I would recommend giving a date matching the July or August monthly release (first Tuesday of the month) to give folks time to make the changes.